Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -849,6 +867,21 @@ define([
             u_dayTextureSplit : function() {
                 return this.properties.dayTextureSplit;
             },
+            u_clippingPlanesLength : function() {
```

Comment:
Ouch - is it possible to combine some of these uniforms?  Should be straight forward for at least a few of them.

CC @lilleyse

Reply:
Same comment - I assume for glTF and 3D tilesets.

Reply:
I could see `u_clippingPlanesEdgeWidth` being stuffed into the alpha channel of `u_clippingPlanesEdgeColor`. After that `u_clippingPlanesLength` can't really be combined into anything except unrelated uniforms.